network-api/env.default

```diff
@@ -16,3 +16,18 @@ BUILD_THROTTLE_SECONDS=900
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=
 SOCIAL_AUTH_LOGIN_REDIRECT_URL=http://localhost:5000/soc/complete/google-oauth2/
+CSP_DEFAULT_SRC='self'
+CSP_SCRIPT_SRC='self' 'sha256-iN7wpJdxHlpujRppkOA8N0+Mzp0ZqZr3lCtxM00Y63c=' cdn.optimizely.com https://www.google-analytics.com/analytics.js
+CSP_STYLE_SRC='self' code.cdn.mozilla.net fonts.googleapis.com
+CSP_IMG_SRC=* data:
+CSP_FONT_SRC='self' fonts.gstatic.com fonts.googleapis.com code.cdn.mozilla.net
+CSP_CONNECT_SRC='self' ws: *.mozilla.org
+CSP_MEDIA_SRC='self'
+CSP_OBJECT_SRC='self'
+CSP_CHILD_SRC='self'
+CSP_FRAME_SRC=
+CSP_BASE_URI=
+CSP_FRAME_ANCESTORS=
+CSP_FORM_ACTION=
+CSP_SANDBOX='allow-scripts'
```

Comment:
sandboxing breaks code in the django admin that depends on localstorage - perhaps we should not enable sandboxing.